analyzers/tests/SonarAnalyzer.UnitTest/Common/MetricsTest.cs

```diff
@@ -495,8 +495,8 @@ public void Complexity()
         [TestMethod]
         public void CognitiveComplexity()
         {
-            var csharpText = System.IO.File.ReadAllText(@"TestCases\CognitiveComplexity.cs");
-            CognitiveComplexity(AnalyzerLanguage.CSharp, csharpText).Should().Be(109);
+            var csharpText = System.IO.File.ReadAllText(@"TestCases\CognitiveComplexity.CSharp9.cs");
```

Comment:
I think we need to add a separate case for C# 9; otherwise `TestCases\CognitiveComplexity.cs` will never be verified.

Reply:
TestCases\CognitiveComplexity.cs is tested within CognitiveComplexity tests. But okay, will add a separate line with the check